boards/cats_rev1Pro/src/config/cats_config.h

```diff
@@ -23,6 +23,7 @@
 #include "util/types.h"
 
 /* Exported types */
+#define CONFIG_VERSION 201
```

Comment:
what does 201 represent?

Maybe it's an overkill but maybe we should use semantic versioning?
https://semver.org/

We will use it for sure in the overall code.

If you do this you can for example have:
#define CONFIG_VERSION_MAJOR 2
#define CONFIG_VERSION_MINOR 1